### PR TITLE
Update /docs readme to include running site

### PR DESCRIPTION
**Pull request by xmudrii**
I've updated /docs README to include how to run site. Directions are from the #366 PR

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -27,3 +27,12 @@ Do not include a title top level header on the document, e.g. `# Title`. The tit
...
 Simply update the associated `.md` markdown file. All documentation should be in complete sentences.
 
+# Testing changes
+
+If you have Jekyll stack installed you can run site locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.  
+Start site by running the following command from the `/docs` directory:
+```
+bundle exec jekyll serve
+```
+Site will be available at `localhost:4000`.
```
Comment (prateekgogia) [inline docs/README.md]: Do we need a make command for this ? May be `make sitetest`, or something.

**Comment by xmudrii** [comment]
‎‎Don't take this as definitive, but as far as I remember `bundle` is long-running command so would it be a good thing for makefile?
If it would be, I'll try too add this.

**Comment by xmudrii** [comment]
@ellenkorbes What do you think about this, should we add `make` command or just pull as it is?

File: docs/README.md [hunk 2]
```diff
@@ -27,3 +27,12 @@ Do not include a title top level header on the document, e.g. `# Title`. The tit
 
 Simply update the associated `.md` markdown file. All documentation should be in complete sentences.
 
+# Testing changes
+
+If you have Jekyll stack installed you can run site locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.
```
Comment (ellenkorbes) [inline docs/README.md]: "you can run site" → "you can run the website" ?

File: docs/README.md [hunk 3]
```diff
@@ -27,3 +27,12 @@ Do not include a title top level header on the document, e.g. `# Title`. The tit
 
 Simply update the associated `.md` markdown file. All documentation should be in complete sentences.
 
+# Testing changes
+
+If you have Jekyll stack installed you can run site locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.  
+Start site by running the following command from the `/docs` directory:
```
Comment (ellenkorbes) [inline docs/README.md]: "Start site by" → "Serve the website by" ?

File: docs/README.md (hunk 1 above)
Comment (ellenkorbes) [inline docs/README.md]: "Site" → "The website" ?

**Comment by ellenkorbes** [comment]
I'm not sure I understand what's the intent here.

If you want to just "compile" the website, `jekyll build` works. If you want to serve it with Jekyll, `jekyll serve` as you suggested. You can serve it however you want though, I often just toss in something like...

```go
package main

import "net/http"

func main() {
	http.Handle("/", http.FileServer(http.Dir("./")))
	http.ListenAndServe(":3000", nil)
}
```

...into the folder depending on what I'm doing.

As for `make` I got no idea what that would do or what purpose it would serve; that's a bit beyond me at this point.

**Comment by xmudrii** [comment]
@ellenkorbes I wanted to include your instructions from the docs (https://github.com/kris-nova/kubicorn/pull/366#issuecomment-326284029) so somebody who wants to test stuff before sending PR, knows how to run the website. I thought it would be useful.

**Comment by ellenkorbes** [comment]
@xmudrii I agree that it is useful, and your instructions are correct. I just have no idea what this `make` talk is about.

**Comment by xmudrii** [comment]
@ellenkorbes Maybe adding `make sitebuild` with `bundle build` would be useful, but I think we should merge this as it is and leave it for another PR.

**Comment by xmudrii** [comment]
@ellenkorbes thanks for the review 😊

**Pull request merged**